components/form/autocompleted/src/index.js

```diff
@@ -127,7 +128,11 @@ export default class FormAutocompleted extends Component {
   }
 
   _renderSubmitButton = ({ text, icon: Icon }) => (
-    <button className='sui-FormAutocompleted-submit' onClick={this._handleSubmit}>
+    <button
+      className='sui-FormAutocompleted-submit'
+      onClick={this._handleSubmit}
+      ref={node => { this.submit = node }}
```

Comment:
could you add `this.submit` and `this.suggestList` to the constructor? (this.submit = null)